pkg/auth/authentication_test.go

```diff
@@ -38,45 +39,35 @@ func assertResultOk(t *testing.T, err error, expectedMsg string) {
 	}
 }
 
-func generateValidTTL(timestamp string) int64 {
-	seconds, _ := strconv.ParseInt(timestamp, 10, 64)
-	current := time.Now().Unix()
-	return (current - seconds) + 1000
-}
-
 var validateCredentialsTc = []validateCredentialsData{
 	{
-		name:          "Valid Credentials",
-		timeToLiveGen: generateValidTTL,
+		name:      "Valid Credentials",
+		tolerance: (time.Now().Unix() - validTimeStamp) + 1000,
```

Comment:
why not just 1000?

Reply:
Saw this comment after y merge the PR. Short answer: Tests depend on a signature that is a constant, that signed message includes a given timestamp, lets say 01/01/2019 00:00:00 000
So the tolerance needs to be, the time that passed from that date till today + the "real tolerance".

This is because the test is bad, i will be working on improving this tests shortly, and this 'hack' wont be necessary

Related [Git Issue ](https://github.com/decentraland/auth-go/issues/17)